modules/ROOT/pages/clustering/monitoring/endpoints.adoc

```diff
@@ -177,21 +178,30 @@ The following table explains how results can be compared.
 [options="header", cols="<1,2,2"]
 |===
 | Name of check                            | Method of calculation                                                                                                                                        | Description
-| `allServersAreHealthy`                     | Every Primaries' status endpoint indicates `isHealthy`==`true`.                                                                                                   | To en sure the data across the entire cluster is healthy.
-                                                                                                                                                                                                            Whenever any Primaries are false that indicates a larger problem.
-| `allVotingSetsAreEqual`                    | For any 2 Primaries (A and B), status endpoint A's `votingMembers`== status endpoint B's `votingMembers`.                                                          | When the voting begins, all the Primaries are equal to each other, and all members agree on membership.
-| `allVotingSetsContainAtLeastTargetCluster` | For all Primaries (*S*), excluding Primary Z (to be switched off), every member in *S* contains *S* in their voting set.
-                                             Membership is determined by using the `memberId` and `votingMembers` from the status endpoint.                                                                   | Sometimes network conditions are not perfect and it may make sense to switch off a different Primary than the one originally was to be switched off.
-                                                                                                                                                                                                            If this check is run for all Primaries, the ones that match this condition can be switched off (providing other conditions are also met).
-| `hasOneLeader`                             | For any 2 Primaries (A and B), `A.leader == B.leader && leader!=null`.                                                                                             | If the leader is different then there may be a partition (alternatively, this could also occur due to bad timing).
+| `allServersAreHealthy`                     | Every primaries' status endpoint indicates `isHealthy`==`true`.                                                                                                   | To en sure the data across the entire cluster is healthy.
```

Comment:
Unrelated to your change, but I happened to notice a rogue space and an out-of-place plural:
```suggestion
| `allServersAreHealthy`                     | Every primary's status endpoint indicates `isHealthy`==`true`.                                                                                                   | To ensure the data across the entire cluster is healthy.
```